### ISSUE TITLE
Polkadot.js wallet auto-closes when trying to access a different extension, such as password manager

### ISSUE THREAD

**Issue by R3dn0ze** [polkadot-js/apps#4934]
This is mostly an issue for those who have password managers and are trying to create a new wallet on polkadot.js wallet extension for chrome (unsure about firefox). When attempting to open a new wallet, a pop-up will show up from the extension, which is normal and expected. Then the issue starts when you try to copy the seedphrase to your password manager within chrome, which is another extension. The issue is that whenever i open my password manager, or click ANYWHERE within chrome that is not the polkadot.js wallet pop-up, the wallet pop-up closes and registration will need to be done from scratch. So my initial thought was like, okay, sure, i'll copy it and paste it later into my password manager.

Obviously this caused a second issue which is when i realised that i needed to create a password for the wallet (obviously). So basically i had to open the password manager again to generate a password which then closed the wallet again. The issue here is that i will need to copy both the seedphrase AND the password simultaneously, which is an issue as you cannot have two different items in the clipboard, to be saved later (without any third-party apps)

I am unsure if there is a way to not have the pop-up close whenever you click anywhere within chrome that is not specifically on the pop-up itself. If not, I have one solution:

- Make the extension open a new tab dedicated for wallet creation, instead of a pop-up. Just for wallet creation, the rest can continue as a pop-up.

TL;DR : Make the polkadot.js extension pop-up not auto-close when trying to access other extensions like password managers in chrome. If not possible due to chrome 'rules', please make the wallet creation open dedicated web tab instead of just a pop-up.

**Comment by Swader** [comment]
The settings window will allow you to open the extension in a new tab: 
![image](https://user-images.githubusercontent.com/1430603/112508401-caf7a880-8d8f-11eb-920d-87a30e33af39.png)

**Comment by jacogr** [comment]
See https://github.com/polkadot-js/extension/issues/472 & https://github.com/polkadot-js/extension/issues/400
